src/components/authentication/configureFirebase.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useCallback } from 'react';
+
+const AuthContext = React.createContext(
+    {
+        token: '',
+        isLoggedIn: true,
+        login: (token) => { },
+        logout: () => { },
+    }
+);
+
+export const AuthContextProvider = (props) => {
+    let initialToken;
+
+    const [token, setToken] = useState(initialToken);
+
+    const userIsLoggedIn = !!token;
+
+    const logoutHandler = useCallback(() => {
+        setToken(null);
+        localStorage.removeItem('token');
+    }, []);
```

Comment:
why empty array? `[]`

Reply:
Will return a memoized version of the callback that only changes if one of the dependencies has changed. That is an array of dependencies working, but in our function, I didn't have any changes.